src/engraving/libmscore/beam.cpp

```diff
@@ -1779,15 +1779,15 @@ void Beam::read(XmlReader& e)
             setGrowLeft(e.readDouble());
         } else if (tag == "growRight") {
             setGrowRight(e.readDouble());
-        } else if (tag == "y1") {
+        } else if (tag == "y1" && score()->mscVersion() >= 300) {
```

Comment:
The following would be slightly more efficient (and maybe more beautiful code):
```c++
} else if (tag == "y1") {
    if (score()->mscVersion() < 300) {
        // before 3.0, custom offset used to be expressed in different coordinate systems, 
        // and thus would mess up the score
        e.skipCurrentElement();
    } else {
       if (fragments.empty()) {
            fragments.push_back(new BeamFragment);
       }
       BeamFragment* f = fragments.back();
       ...
```
Because that way you always stop comparing the tag if the it matches "y1". Otherwise, you do a lot of unnecessary comparisons in the else-ifs after "y1".